client/resultsview/table.js

```diff
@@ -240,6 +240,34 @@ var TableView = Element.View.extend({
         this.$columnHeaderRow.html(html);
 
         html = '';
+        
```

Comment:
OK, so this is what it looks like for me:

<img width="493" alt="screen shot 2016-07-15 at 18 43 10" src="https://cloud.githubusercontent.com/assets/3240247/16868618/02ccd9b8-4abc-11e6-81f9-2a6e8b728d7d.png">

which is almost right, it's just the header that isn't correct. but that's because you swap the header around *after* it's already been inserted into the HTML. if you look at the lines of code 234-237, that's where the header is inserted. so i'd shift this block of code up to, say, line 231, and it should all work.